### PR TITLE
Support sampling/filtering of span logs using updated span log data f…

### PR DESCRIPTION
**Pull request by hanwavefront**
…ormat

### CODE REVIEW

**Comment by basilisk487** [comment]
LGTM (pending java-lib release)

File: proxy/src/main/java/com/wavefront/agent/listeners/tracing/TracePortUnificationHandler.java [hunk 1]
```diff
@@ -199,6 +197,76 @@ public static void preprocessAndHandleSpan(
...
+              handler.reject(spanLogs, spanMessageHolder[0]);
+            } else {
+              handler.block(spanLogs);
+            }
+            return;
+          }
+        }
+        List<Span> spanOutput = new ArrayList<>(1);
+        try {
+          spanDecoder.decode(spanMessage, spanOutput, "dummy");
```
Comment (srujann) [inline proxy/src/main/java/com/wavefront/agent/listeners/tracing/TracePortUnificationHandler.java:240]: `dummy`?

Comment (basilisk487) [reply proxy/src/main/java/com/wavefront/agent/listeners/tracing/TracePortUnificationHandler.java:240]: it doesn't really matter in this context, but we should probably standardize on something like "default" or "unknown"

Comment (hanwavefront) [reply proxy/src/main/java/com/wavefront/agent/listeners/tracing/TracePortUnificationHandler.java:240]: yeah, right now i think we're defaulting to "dummy" in most places in the code. let me know if i should change that

**Pull request merged**